### DISCUSSION TITLE
Jetson Nano devkit (sdcard) root device error

### DISCUSSION THREAD

**Discussion by tirankas** [OE4T/tegra-demo-distro#107] [General]
Hello Guys,
Have you seens this before:

[    5.519208]  mmcblk1: p1 p2 p3
[    5.522767] Waiting for root device /dev/mmcblk0p1...

It waits forever so I assume the block device index is wrong.
Should I set it to mmcblk1? Looking at the tegra demo, I dont see this set anywhere and the default is 0.

Thanks

**Comment by tirankas** [comment]
Fixed, next issue is:
cat: can't open '/sys/block/mmcblk0/size': No such file or directory

For some reason the mender still points to mmcblk0

**Reply by madisongh** [reply]
I think there's a variable setting for that.

**Reply by ichergui** [reply]
Hey @tirankas 
Could you please share the steps that you taking for the build ?
It should be working fine

**Comment by tirankas** [comment]
It was no issue at the end of the day...
Cleaning mender related recipes and rebuilding, everything works fine
